lib/internal/webstreams/readablestream.js

```diff
@@ -654,13 +655,8 @@ class ReadableStreamBYOBRequest {
         'This BYOB request has been invalidated');
     }
 
-    const viewByteLength = ArrayBufferViewGetByteLength(view);
-    const viewBuffer = ArrayBufferViewGetBuffer(view);
-    const viewBufferByteLength = ArrayBufferGetByteLength(viewBuffer);
-
-    if (viewByteLength === 0 || viewBufferByteLength === 0) {
-      throw new ERR_INVALID_STATE.TypeError(
-        'View ArrayBuffer is zero-length or detached');
+    if (isViewedArrayBufferDetached(view)) {
+      throw new ERR_INVALID_STATE.TypeError('Viewed ArrayBuffer is detached');
```

Comment:
> Assert: this.[[view]].[[ByteLength]] > 0.
> Assert: this.[[view]].[[ViewedArrayBuffer]].[[ByteLength]] > 0.

Are we guaranteed that these assertions never fail?

Reply:
Fixed.